src/System.Private.CoreLib/shared/System/StringComparer.cs

```diff
@@ -159,9 +160,9 @@ public int GetHashCode(object obj)
             return obj.GetHashCode();
         }
 
-        public abstract int Compare(string x, string y);
-        public abstract bool Equals(string x, string y);
-        public abstract int GetHashCode(string obj);
+        public abstract int Compare(string? x, string? y);
+        public abstract bool Equals(string? x, string? y);
+        public abstract int GetHashCode(string? obj);
```

Comment:
TODO-NULLABLE? `GetHashCode` is not intent to accept nulls, right?

Reply:
Commented.